### DISCUSSION TITLE
logical Kotlin Question Answer

### DISCUSSION THREAD

**Discussion by dimpeshpanwar** [dimpeshpanwar/KOTLIN-Programs#11] [Q&A] [answered]
you have filed logical Q&A

**Comment by Rishi098** [comment, marked as answer]
Write a function that checks whether a given number is prime.

fun isPrime(num: Int): Boolean {
    if (num <= 1) return false
    for (i in 2 until num) {
        if (num % i == 0) return false
    }
    return true
}
